docs/typedef/modules/controllers.js

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This file should not be in an `api` directory, not `modules`.

Reply:
will move it, after discussion below is solved